README.md

```diff
@@ -1,60 +1,63 @@
-# Introduction #
+<p align="center">
+  <img width="460" src="doc/logo/psyclone_v1.0.png">
+</p>
 
-Welcome to PSyclone. PSyclone is a code generation and transformation
-system that generates appropriate code for the PSyKAl code structure
-developed in the GungHo project. It is also capable of working with
-existing Fortran code.
+![Build Status](https://github.com/stfc/PSyclone/workflows/PSyclone%20tests%20and%20examples/badge.svg)
+[![codecov](https://codecov.io/gh/stfc/PSyclone/branch/master/graph/badge.svg)](https://codecov.io/gh/stfc/PSyclone)
 
-Please see [psyclone.pdf](psyclone.pdf) in this directory (or on
-[ReadTheDocs](http://psyclone.readthedocs.io)) for
-more information. If you would prefer to build the documentation,
-please see the [README](doc/README.md) file in the "doc" directory.
+PSyclone is a source-to-source Fortran compiler designed to programmatically
+optimise and parallelise HPC applications using user-provided transformation
+scripts. These scripts offer a separation of concerns between the science code
+and the performance portability implementation choices, allowing to experiment
+and develop with each side independenly. Additonally, PSyclone supports the
+development of kernel-based Fortran-embedded DSLs following the PSyKAl model
+developed in the [GungHo project](https://www.metoffice.gov.uk/research/foundation/dynamics/next-generation).
 
-# Try it on Binder #
+PSyclone is currently used to support the
+[LFRic PSyKAl mixed finete elements DSL](https://www.metoffice.gov.uk/research/modelling-systems/lfric/)
```

Comment:
"finite" and "-element"

Reply:
Possibly a link to the relevant bit of the User Guide would be good here too?

Reply:
Done, then I move the LFRic link to the "UK MetOffice's next generation modelling system" text, and I followed a similar pattern for GOcean.